ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/BehaviorTreeRootNodeExecutor.java

```diff
@@ -120,16 +176,18 @@ else if (state.getActionChildren().get(i).calculateExecuteAfterActionIndex() < e
             state.getLogger().info("End of sequence.");
             state.setAutomaticExecution(false);
          }
-         else if (anyActionExecutionFailed)
+         else if (!failedActionsWithoutFallback.isEmpty())
          {
-            state.getLogger().error("An action failed. Disabling automatic execution.");
+            state.getLogger().error("An action failed. Disabling automatic execution.\n   Failed: %s".formatted(failedActionsWithoutFallback));
             state.setAutomaticExecution(false);
          }
          else
          {
             while (shouldExecuteNextAction())
             {
-               state.getLogger().info("Automatically executing action: {}", actionChildren.get(state.getExecutionNextIndex()).getClass().getSimpleName());
+               var nextAction = actionChildren.get(state.getExecutionNextIndex());
```

Comment:
Hmm `var`? That's cool. We don't have any style guidelines against it, do we?

Reply:
I think it's totally fine in places like this where the type is clunky and not important for understanding what's going on